libs/ledger-live-common/src/families/aptos/bridge.integration.test.ts

```diff
@@ -64,10 +64,8 @@ const aptos: CurrenciesData<Transaction> = {
             family: "aptos",
             mode: "send",
             fees: "1100",
-            options: '{ "maxGasAmount": "11", "gasUnitPrice": "100" }',
-            estimate:
+            options:
               '{ "maxGasAmount": "11", "gasUnitPrice": "100", "sequenceNumber": "1", "expirationTimestampSecs": "1734535375" }',
```

Comment:
wasn't `sequenceNumber` and `expirationTimestampSecs` removed?

Reply:
this is a good point. If it was removed, how did it not failed? 🤔